aws/iam.go

```diff
@@ -0,0 +1,13 @@
+package aws
+
+import (
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/iam"
+	"github.com/gruntwork-io/cloud-nuke/config"
+)
+
+func getAllIamRoles(session *session.Session, input *iam.ListRolesInput) ([]*string, error) {}
```

Comment:
Is it necessary to take in the input? Or can it create that on the fly within the function?